### DISCUSSION TITLE
Yellow PCBs shipping from belgium

### DISCUSSION THREAD

**Discussion by fparie** [FundamentalFrequency/.github#33] [General]
I have three yellow PBCs (my mistake, you can barely see the lemons) 

Yours for shipping cost
Shipping in belgium is 5.70 EUR
Shipping to most of europe 10 EUR
if you want tracking it'll be more 
https://www.bpost.be/en/send-parcel/international

**Comment by danilogiacomi** [comment]
Hi, I'm in Italy and I'd like one if they're available! :)

**Reply by fparie** [reply]
i'm on discord, drVonDouchFace for details

**Comment by P1NTU** [comment]
Hi, I'm in the UK i'd like the other two if they are still available

**Reply by fparie** [reply]
i'm on discord, drVonDouchFace for details

**Comment by muldvar** [comment]
hey i was wondering if you still had one for shipping to NL?

**Comment by seanarenouf** [comment]
hey OP - any left?
